File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let numbers = []
        for (let i = 0; i < nums.length; i++) {
            if (numbers.includes(nums[i])) {
                return true;
            }
            numbers.push(nums[i])
        }
        return false;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int x = 0, y = numbers.length - 1;
        while (x < y) {
            int total = numbers[x] + numbers[y];
            if (total > target) {
                y--;
            } else if (total < target) {
                x++;
            } else {
                return new int[]{x+1, y+1};
            }
        }
        return new int[0];
    }
}
